api_fhir_r4/converters/organisationConverter.py

```diff
@@ -0,0 +1,212 @@
+from django.utils.translation import gettext
+from policyholder.models import PolicyHolder
```

Comment:
@dragos-dobre , this is not a issue for me but that mean that PH is a dependency of FHIR R4